spec/features/admin/overview_spec.rb

```diff
@@ -123,5 +110,45 @@
         end
       end
     end
+
+    context "with the spree dash configured" do
+      let(:d1) { create(:distributor_enterprise) }
+
+      before do
+        stub_jirafe
+        @enterprise_user.enterprise_roles.build(enterprise: d1).save
+      end
+
+      around do |example|
+        with_dash_configured { example.run }
```

Comment:
Aah, good question. Without knowing the details of which environments were breaking, I believe that's what was going on. Even if we're not using it, I think it'll be good to have this here so that if someone does enable dash, they don't get mysterious errors.